mongonaut/views.py

```diff
@@ -300,7 +297,7 @@ def get_context_data(self, **kwargs):
 
         return context
 
-    def get_form(self, Form):
+    def get_form(self):
```

Comment:
Before line `303` was getting passed a dynamic form. It is now always using the default `Form` why the change? Is it no longer passed a specific dynamic form?